docs/website/docs/walkthroughs/configure-a-pipeline.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Configure a pipeline
+description: How to configure a pipeline
+keywords: [how to, configure a pipeline]
+---
+
+# Configure a pipeline
+
+To configure your pipeline in `dlt`, you can use providers to set up your configurations. These configurations can be specified using one or a combination of the following methods:
+
+1. **Environment Variables:** Set configuration parameters directly in your environment, which `dlt` can read at runtime.
+2. **Vaults:** Use secure vault services such as Airflow, Google Cloud, AWS, or Azure to manage and retrieve configuration parameters.
+3. **Configuration Files:** Utilize `secrets.toml` and `config.toml` files to define your pipeline settings.
+4. **Default Argument Values:** Specify default values directly within your code when initializing the pipeline.
+
+## Example Configuration
+
+To configure a `dataset_name_layout` which will prefix the names of all datasets created by your pipeline, follow these steps using one of the following methods:
+
+
+1. **Environment Variables:**
```

Comment:
why not have one example for each of the points listed above in the same order that sets the same variable.